Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -87,6 +86,7 @@ public void initializeNetwork(@NonNull final Context context, @Nullable final Ma
             try {
                 if (Vungle.isInitialized()) {
                     networkInitializationSucceeded = true;
+                    sVungleRouter.setsInitState(VungleRouter.SDKInitState.INITIALIZED);
```

Comment:
are we sure about the state? I believe in adapters we are considering that Vungle can become de-inited for some resons

Reply:
Here only logic being changed is while during initialization  of Vungle adapter, when we check for SDK init directly from SDK to update the router class, so that when individual adapter instances dont again have to call Vungle.init. As it does not make sense, why we skipped that part in VungleAdapterConfiguration but will continue to do from the adapters class.
Reg. Vungle/SDK possibly being de-initialized part, from adapters side once VungleRouter is initialized once we anyways for future are not checking status from SDK, so my changes do not really affect the overall initialized status from the overall SDK session duration perspective. If we want to do that  then we will have to remove the initialized check from VungleRouter and always depend on  the SDK isInitialized